packages/nodes-base/utils/utilities.ts

```diff
@@ -217,6 +217,34 @@ export const keysToLowercase = <T>(headers: T) => {
 	}, {} as IDataObject);
 };
 
+/**
+ * Formats a private key by removing unnecessary whitespace and adding line breaks.
+ * @param privateKey - The private key to format.
+ * @returns The formatted private key.
+ */
+export function formatPrivateKey(privateKey: string): string {
```

Comment:
In general we should be moving code out of this file, and into `this.helpers`. That way these utility functions are also available 3rd-party nodes.

Reply:
@netroy that isn't a bad idea, Fancy making a tech debt ticket for it and we can move them all out on our next node team tech debt day